Assets/Scripts/Managers/SaveFile.cs

```diff
@@ -23,6 +23,7 @@ public struct QuestDetails
         public string name;
         public int turnsLeft;
         public int cost;
+        public string reward;
```

Comment:
This probably doesn't need to be serialised since it will be preconfigured by the quest